python/semantic_kernel/connectors/search_engine/bing_connector.py

```diff
@@ -20,12 +20,16 @@ class BingConnector(ConnectorBase):
 
     _api_key: str
 
-    def __init__(self, api_key: str | None = None, env_file_path: str | None = None) -> None:
+    def __init__(
+        self, api_key: str | None = None, custom_config: str | None = None, env_file_path: str | None = None
+    ) -> None:
         """Initializes a new instance of the BingConnector class.
 
         Arguments:
             api_key {str | None}: The Bing Search API key. If provided, will override
                 the value in the env vars or .env file.
+            custom_config {str | None}: The Bing Custom Search instance's unique identifier.
+                If provided, will overide the value in the env vars or .env file.
```

Comment:
@bochris quick call out to fix the typo, please. Should be `override`.

Reply:
good catch, fixed!